src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,6 +97,31 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
+		if ( 'scheduled' === $this->tab->get_slug() && isset( $_REQUEST['s'] ) && ! empty( $_REQUEST['s'] ) ) {
+			// nonce check if search form submitted
+			$nonce_check = isset( $_POST['s'] ) && isset( $_POST['aggregator']['nonce'] ) ? wp_verify_nonce( $_POST['aggregator']['nonce'], 'aggregator_' . $this->tab->get_slug() . '_request' ) : 1;
+			if( $nonce_check !== false ) {
+				$args['meta_query'] = [
```

Comment:
:warning: Detected usage of meta_query, possible slow query.

<sub>hash: d21854879827952ff4fb5b9d21e24513</sub>

Reply:
irrelevant. meta_query is necessary when search query is posted.